src/log.jl

```diff
@@ -33,36 +33,58 @@ function logging_summary(m::PODNonlinearModel)
+        println("number of continuous variables = ", length([i for i in 1:m.num_var_orig if m.var_type[i] == :Cont]))
+        println("number of binary variables = ", length([i for i in 1:m.num_var_orig if m.var_type[i] == :Bin]))
+        println("number of integer variables = ", length([i for i in 1:m.num_var_orig if m.var_type[i] == :Int]))
+
+        println("detected nonlinear terms = ", length(m.nonconvex_terms))
+        for i in POD_C_NLTERMS
+            cnt = length([1 for j in keys(m.nonconvex_terms) if m.nonconvex_terms[j][:nonlinear_type] == i])
+            cnt > 0 && println("\tTerm $(i) Count = $(cnt) ")
+        end
+        println("number of variables involved in nonlinear terms = ", length(m.candidate_disc_vars))
+        println("number of selected variables to discretize = ", length(m.disc_vars))
 
-        m.minlp_local_solver != UnsetSolver() && println("MINLP local solver = ", split(string(m.minlp_local_solver),".")[1])
-        println("NLP local solver = ", split(string(m.nlp_local_solver),".")[1])
+        println("* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *")
+        println("                        SUB-SOLVERS  ")
+        m.minlp_solver != UnsetSolver() && println("MINLP local solver = ", split(string(m.minlp_solver),".")[1])
+        println("NLP local solver = ", split(string(m.nlp_solver),".")[1])
         println("MIP solver = ", split(string(m.mip_solver),".")[1])
-
+        println("* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *")
+        println("                    SOLVER CONFIGURATION ")
         println("maximum solution time = ", m.timeout)
         println("maximum iterations =  ", m.maxiter)
         @printf "relative optimality gap criteria = %.5f (%.4f %%)\n" m.relgap (m.relgap*100)
-        println("detected nonlinear terms = $(length(m.nonlinear_terms))")
-        println("number of variables involved in nonlinear terms = $(length(m.all_nonlinear_vars))")
-        println("number of selected variables to discretize = $(length(m.var_disc_mip))")
-
-        m.bilinear_convexhull && println("bilinear treatment = convex hull formulation")
-        m.monomial_convexhull && println("monomial treatment = convex hull formulation")
-
-        println("using piece-wise convex hull : $(m.convhull_formulation) formulation")
-        println("using method $(m.disc_var_pick) for picking discretization variable...")
-
+        println("default tolerance = ", m.tol)
+        m.recognize_convex && println("actively recognize convex patterns = ")
+        println("basic bound propagation = ", m.presolve_bp)
+        println("use piece-wise relaxation formulation on integer variables = ", m.int_enable)
+        println("piece-wise relaxation formulation = $(m.convhull_formulation) formulation")
+        println("method for picking discretization variable = $(m.disc_var_pick)")
+        println("conseuctive solution rejection = after ", m.disc_consecutive_forbid, " times")
```

Comment:
consecutive*

Reply:
Thanks. This is addressed in the most recent commit.